app/results.py

```diff
@@ -188,8 +188,13 @@ def get(self):
             # If a query looks like a phone number, show the user a result
             # of looking up the number in the carriers-provided BBS system.
             if self.config.jp_mobile_carrier_redirect:
-                if jp_mobile_carriers.handle_phone_number(self, query.query):
-                    return 
+                try:
+                    if jp_mobile_carriers.handle_phone_number(self, query.query):
+                        return
+                except Exception as e:
+                    logging.error('failed to scrape search result for the phone number.')
```

Comment:
Over 80 chars.

FYI We are following the external version of Python Style Guide:
https://google.github.io/styleguide/pyguide.html
which is a bit different from the internal one.